c7n/filters/offhours.py

```diff
@@ -164,7 +164,6 @@ class Time(Filter):
             'weekends': {'type': 'boolean'},
             'weekends-only': {'type': 'boolean'},
             'opt-out': {'type': 'boolean'},
-            'debug': {'type': 'boolean'},
```

Comment:
note, this wasn't used anywhere afaict. just a dead reference from some time long ago?